README.md

```diff
@@ -59,7 +59,16 @@ There are three ways to do this:
 
 ## Examples
 
-The [`examples`](https://github.com/ni/measurementlink-python/tree/main/examples/) directory contains example measurement services. See the [README.md](https://github.com/ni/measurementlink-python/tree/main/examples/README.md) file for more information.
+Example measurements are available in the `examples` directory of the `measurementlink-python` Git repository. You can download the examples by cloning the Git repository or downloading the appropriate source code zip file.
+
+> **Warning**
```

Comment:
While I agree with this warning, it is pretty strange. I wonder if we could do development in a private fork (or even a separate, public branch) and then push it public when the next version of IS and ML release. That might be a terrible idea, but I also don't like the idea of our mainline containing changes that customers can't use.

Reply:
I agree that this seems best. That aside, we typically use Caution for physical injury and Warning for data loss, so in writer-owned docs this would be a Note or Info. You could just drop the label and leave the text.

Reply:
I added a release asset containing a zip file of examples, changed these docs to directly link to the zip file, reworded the warning, and removed the warning label.